### ISSUE TITLE
[Feature request] Ability to copy commit SHA in file history screen itself

### ISSUE THREAD

**Issue by abhi3700** [ForkIssues/Tracker#595]
When I open file history, another screen opens up.
But currently, in order to copy the SHA (by right click), we go back to the main repository screen on clicking that block.

Then, again we have to open the file history in order to copy another SHA of file on another commit.

So, please consider this feature in the next update i.e. 
__When we right click on any row in file history screen, it shows the option to copy the SHA at that point itself. Basically, no need to go back to that commit in main screen (where so many other files in that commit is available)__

I hope you get the feature request.

**Comment by DanPristupov** [comment]
What is your workflow? I mean, what do you do with those SHA's afterwards?

**Comment by abhi3700** [comment]
Actually, I want to see excel diff via batch file.

So, I open batch file and I enter the commits & see comparison using `spreadsheetcompare` in MS Office 2013 or newer.

I was thinking if there is any option of copying commit from Fork Client.

**Comment by abhi3700** [comment]
@DanPristupov Is there any way out for my problem or can I expect this feature in the next release?

**Comment by olets** [comment]
If I understand your use case, you want a list of the hashes for all the commits that changed a certain file. To me that sounds like a job for the command line, not a Git UI. You can do that with this:

```shell
git log --pretty=format:"%H" --follow <file name>
```

For example

```shell
git log --pretty=format:"%H" --follow README.md
```

**Comment by abhi3700** [comment]
Good news!!...
It has been implemented in the current version of `Git-Fork v1.38.0.0`

**Comment by abhi3700** [comment]
Good news!!...
It has been implemented in the current version of `Git-Fork v1.38.0.0`
